src/components/modal/__stories__/modal.stories.ts

```diff
@@ -3,73 +3,99 @@
  */
  
 import { html } from 'lit';
-
+import { createOptionsArray } from '../../../global/mixins/global';
 import { PREFIX_TAG } from '../../../global/settings/settings';
-import '../modal';
+import { ICON_IDS } from '../../../global/defs/iconIds';
+import { BUTTON_TYPES, BUTTON_ICON_POSITION } from '../../button/defs';
 import '../../button/button';
+import { MODAL_SIZES } from '../defs';
+import '../modal';
+
+const optionsModalSize = createOptionsArray(MODAL_SIZES);
 
 export default {
   title: 'Web Components/Modal',
   component: `${PREFIX_TAG}-modal`,
   argTypes: {
     size: {
-      options: ['default', 'large', 'small'],
-      control: { type: 'radio' },
+      description: 'Size of the modal window.',
+      options: [...optionsModalSize],
+      control: { type: 'select' },
     },
-    label: {
-      control: { type: 'text' },
-    },
-    content: {
+    closeLabel: {
```

Comment:
Don't make this a settable parameter. Just hard code to "Close window" for now.

Reply:
Why do you want to hardcode this? It'll need to be localized so this provides a way to set that value.

Reply:
I updated the default value to "Close window".